pnpm-workspace.yaml

```
@@ -0,0 +1,13 @@
+packages:
+  - /*
+  - packages/*
```

The `/*` path causes that pnpm scans all directories in the filesystem root. On Windows, it ends with `EPERM: operation not permitted, scandir 'D:\System Volume Information'` to which pnpm has no access.

It seems that the root package is always included, even when custom location wildcards are used.

```suggestion
  - packages/*
```

---

Fixed